libkbfs/disk_block_cache.go

```diff
@@ -667,9 +665,17 @@ func (*DiskBlockCacheStandard) getRandomBlockID(numElements,
 	return kbfsblock.MakeRandomIDInRange(0, pivot)
 }
 
+// evictSomeBlocks tries to evict <numBlocks> blocks from the cache. If
```

Comment:
Can we standardize on markdown ticks for variable names in comments?  i.e. \`numBlocks\`.  (And \`blockIDs\` on the next line.)